src/libsync/syncjournaldb.cpp

```diff
@@ -286,6 +286,13 @@ bool SyncJournalDb::checkConnect()
         return sqlFail("Create table version", createQuery);
     }
 
+    // create the checksumtype table.
+    createQuery.prepare("CREATE TABLE IF NOT EXISTS datafingerprint("
+                               "fingerprint TEXT UNIQUE"
+                               ");");
```

Comment:
I found the @moscicki discussion.. https://github.com/owncloud/client/issues/3742